### PR TITLE
3E Events connector

### PR DESCRIPTION
**Pull request by isalzh**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

### CODE REVIEW

**Comment by v-EgorKozhadei** [comment]
Hello @isalzh, 

Thank you very much and congratulations on your new submission. 
Please sign the CLA, we can't start the review process without it. You can see the link above this message.
Once it's signed, our Certification Engineers will review the PR within 1-2 weeks. 

Please let us know if you have any questions. 

Thank you very much again for working with us.

**Comment by v-EgorKozhadei** [comment]
Hello @isalzh, 

I hope you are doing well. 

I am following up with you to see how things are going with preparing your submission. 
We can't start the review without a signed CLA. You can find it above my messages.

Please let us know if you have any questions. 

Thank you.

File: certified-connectors/3E Events/readme.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+[[_TOC_]]
```
Comment (sriyen-msft) [inline certified-connectors/3E Events/readme.md:1]: @isalzh Can you please let us know the purpose of the above line?

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/3E Events/readme.md (hunk 1 above)
Comment (isalzh) [reply certified-connectors/3E Events/readme.md:1]: It's table of contents markdown. https://docs.microsoft.com/en-us/azure/devops/project/wiki/wiki-markdown-guidance?view=azure-devops
Looks like it's not supported by GitHub and we missed that fact. Fill fix, thank you.

Comment (sriyen-msft) [reply certified-connectors/3E Events/readme.md:1]: @isalzh Please follow the template we have provided for `"readme.md"` file and update this file by removing the TOC line.

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/3E Events/readme.md (hunk 1 above)
Comment (isalzh) [reply certified-connectors/3E Events/readme.md]: removed

**Review by sriyen-msft — APPROVED** [review]

**Comment by sriyen-msft** [comment]
@isalzh You need to sign the CLA to get this PR merged into our repo.

**Comment by v-EgorKozhadei** [comment]
Hello @isalzh, 

I hope you are doing well. 

Please sign the CLA, we can't merge the pull request without it. You can see it above all the messages or [follow this link to see it](https://github.com/microsoft/PowerPlatformConnectors/pull/1830#issuecomment-1192595561). 

Please let us know if you have any questions. 

Thank you.

**Comment by v-EgorKozhadei** [comment]
Hello @isalzh, 

I hope you are doing well. 

Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-updates) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file. 

After making the submission in ISV Studio, please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6. 

Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector.  

We expect all tests to be completed within 1-2 weeks. 

If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions. 

Please let me know if you have any questions. 

Thank you very much for working with us.

**Pull request merged**